<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scrapyard</class>
 <widget class="QDialog" name="Scrapyard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Points</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pointSpin"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="typeWeaponRadio">
         <property name="text">
          <string>Weapons</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optTypeRadioGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeLaunchRadio">
         <property name="text">
          <string>Launch</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optTypeRadioGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeStatRadio">
         <property name="text">
          <string>Stat</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optTypeRadioGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="typeSpecialRadio">
         <property name="text">
          <string>Special</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optTypeRadioGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="broadsideCheck">
         <property name="text">
          <string>Broadside</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="singleCheck">
         <property name="text">
          <string>One Only</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="weaponBox">
         <item>
          <layout class="QHBoxLayout" name="weaponButtons">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="newWeaponButton">
                 <property name="text">
                  <string>New</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteWeaponButton">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="editWeaponbutton">
                 <property name="text">
                  <string>Edit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="CopyWeaponButton">
                 <property name="text">
                  <string>Copy</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="weaponLayout"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="launchBox_2">
         <item>
          <layout class="QHBoxLayout" name="launchAssetLayout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="launchAddButton">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="launchDeleteButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Limited</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="limitedSpin"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="launchQuantitySpin"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="launchProfileLayout"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="statBox">
         <item>
          <widget class="QRadioButton" name="scanRadio">
           <property name="text">
            <string>Scan</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statTypeRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sigRadio">
           <property name="text">
            <string>Signature</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statTypeRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="thrustRadio">
           <property name="text">
            <string>Thrust</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statTypeRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="hullRadio">
           <property name="text">
            <string>Hull</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statTypeRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="pdRadio">
           <property name="text">
            <string>PD</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statTypeRadioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="statBonusSpin"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="specialBox">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="specialAddButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="specialDeleteButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QComboBox" name="specialSelectCombo"/>
           </item>
           <item>
            <widget class="QSpinBox" name="specialNumSpin"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="specialRosterLayout"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="statTypeRadioGroup"/>
  <buttongroup name="optTypeRadioGroup"/>
 </buttongroups>
</ui>
